code/game/objects/structures/signs/signs_interactive.dm

```diff
@@ -111,10 +111,18 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/delamination_counter, 32)
 /obj/structure/sign/collision_counter/Destroy()
 	return ..()
 
-/obj/structure/sign/collision_counter/proc/new_hit()
+/obj/structure/sign/collision_counter/proc/new_hit(lift_master, collided_type)
 	SIGNAL_HANDLER
 
+	if(!ismob(collided_type))
+		return
+
+	var/mob/living/carbon/human/victim = collided_type // Real players only, no gaming high score
```

Comment:
actually I should've made you change the typepath to just ``mob`` because that's where client is and we are expecting more than just humans to be hit, so this is misleading even if it works fine in-game.

Reply:
I'll put it in a no gbp fix

Reply:
https://github.com/tgstation/tgstation/pull/72896